include/cuco/detail/trie/bit_vector/bit_vector_ref.inl

```diff
@@ -202,6 +202,37 @@ class operator_impl<op::bv_read_tag, bit_vector_ref<StorageRef, Operators...>> {
   }
 };
 
+template <typename StorageRef, typename... Operators>
+class operator_impl<op::bv_set_tag, bit_vector_ref<StorageRef, Operators...>> {
+  using ref_type                = bit_vector_ref<StorageRef, Operators...>;  ///< Bitvector ref type
+  using size_type               = typename StorageRef::size_type;            ///< Size type
+  using slot_type               = typename StorageRef::value_type;           ///< Slot type
+  const size_type bits_per_word = sizeof(slot_type) * 8;
```

Comment:
There's one catch with these mixin operators: You cannot have any runtime members in the operator class, or the `static_cast<ref_type&>(*this)` will land you in the middle of an ill-formed cast expression. We're casting this operator to the `ref_type`, and after doing that, the operator's members occupy the same space as the `ref_type` members.
In this particular case, you can just declare `bits_per_word` as `constexpr` to get rid of the problem.